### ISSUE TITLE
SQL Connection times out in Backend

### ISSUE THREAD

**Issue by samstifter** [samstifter/verbose-sniffle#1]

**Comment by distracted-dreamer** [comment]
Switched from using Nodemon to Forever(https://github.com/foreverjs/forever)
Forever reloads automatically after a crash regardless of the reason for the crash. This is not a fix, however, it should make a backend crash not break it until we force a reload.

**Comment by samstifter** [comment]
Yeah I suppose that's a temporary fix at least lol. We should really permanently fix this...
